IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGroundReinforcementElement/DocEntity.xml

```diff
@@ -1,14 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcGroundReinforcementElement" UniqueId="eff9d345-72ca-4b29-8857-cc24f5c55fb0" BaseDefinition="IfcBuiltElement" EntityFlags="32">
 	<Attributes>
-		<DocAttribute Name="PredefinedType" UniqueId="5bf261d3-adfb-4e71-89d3-59707720d1ca" DefinedType="IfcGroundReinforcementElementTypeEnum">
+		<DocAttribute Name="PredefinedType" UniqueId="5bf261d3-adfb-4e71-89d3-59707720d1ca" DefinedType="IfcGroundReinforcementElementTypeEnum" AttributeFlags="1">
 			<Documentation>Specifies the type for which the value is selected from a predefined type enumeration. This type may associate additional specific property sets.</Documentation>
 		</DocAttribute>
 	</Attributes>
 	<WhereRules>
-		<DocWhereRule Name="HasObjectType" UniqueId="566bf0f8-aaeb-41ca-8145-127664b1dbee">
-			<Documentation>The attribute ObjectType shall be given if the predefined type is set to USERDEFINED.</Documentation>
-			<Expression>(PredefinedType &lt;&gt; IfcGroundReinforcementElementTypeEnum.USERDEFINED) OR EXISTS(SELF\IfcObject.ObjectType)</Expression>
+		<DocWhereRule Name="CorrectPredefinedType" UniqueId="7c11e86a-7d74-4f61-b467-9095ec731b2e">
+			<Documentation>Either the _PredefinedType_ attribute is unset(e.g. because an _IfcGroundReinforcementElementType_ is associated), or the inherited attribute _ObjectType_ shall be provided, if the _PredefinedType_ is set to USERDEFINED.</Documentation>
```

Comment:
```suggestion
			<Documentation>Either the _PredefinedType_ attribute is unset (e.g. because an _IfcGroundReinforcementElementType_ is associated), or the inherited attribute _ObjectType_ shall be provided, if the _PredefinedType_ is set to USERDEFINED.</Documentation>
```